app/counselor/views.py

```diff
@@ -153,15 +153,15 @@ def checklist():
         #create new checklist item from form data
         new_item = ChecklistItem(
                     text=form.item_text.data,
-                    assignee_id=current_user.id,
+                    assignee_id=0, #denotes counselor-created item
```

Comment:
Hm... there is a user with id = 0

How are we going to differentiate those, or does it not matter?

Reply:
Actually there isn't, I'm pretty sure (we 1-index for ids)!

Reply:
Cool 👍